RELNKS-QA/index.json

```diff
@@ -701,11 +725,15 @@
+        "css": " .logo-container{background-color: #dcd59a;} #page-login #logo {background-size: contain; background-color: #dcd59a;} #page-login #login { background-color: #af272f; }  .page-HeaderPage>.drawer-layout>.header-wrapper, .page-HeaderPage>.drawer-layout>.content>.padding-wrapper.footer {background-color: #af272f !important;} .theme-accent-bg{background-color: #af272f !important; color:white !important; } .selected .theme-accent-bg{background-color: #147ec8 !important; } .theme-accent-bg.clickable:active,.theme-accent-bg[active] {color: #af272f !important; background-color:white !important; } .theme-accent-fg{color: #af272f !important; } .theme-accent-fg.selected{color: #147ec8 !important; background-color:white !important; } .theme-accent-fg.clickable:active,.theme-accent-fg[active]{background-color: #af272f !important; color:white !important; } .page-HeaderPage>.header {background-color: #af272f !important;} div[page].page-Menu .menu-body {background-color: #af272f;} #page-WelcomePage .content{background-image: radial-gradient(circle, #147ec8, #1070b3, #0d629f, #09558b, #054877);} .fieldgroup input, .fieldgroup .input, .fieldgroup select, .fieldgroup select-multiple, .fieldgroup textarea {  } .huge-buttons button i:first-child, #page-BaseDashItemsPage #dashboard .d_item .d_icon i { background-color: #af272f !important; } .btn-primary, button.primary { background-color: #af272f !important; border: 1px solid #af272f !important;} button.primary:active { background-color: #649623 !important; border-color: #af272f !important; } #page-login #login button.btn_login {border: 1px solid #cccccc; background-color: #cecece  !important; color: #6b6b6b; background-image: none !important;} #page-login #login button.btn_register { background-color: #6b6b6b !important; background-image: none !important; border-color: #969696 !important; color: #cacaca; !important; } #page-login #login button:active { color: #AAA; background-color: #6b6b6b !important; background-image: none !important; border-color: #969696 !important; } .fieldgroup label.toggle input:checked+.radio:before { background-color: #af272f !important; } .fieldgroup label.radio input:checked+.radio, .fieldgroup label.toggle input:checked+.radio { border-color: #af272f !important; } div[type='info-tile-component'] button { background-color: #af272f !important;}#page-DocumentsRepositoryPage .box-info .title-row { color: #af272f !important;}"
```

Comment:
Looks like in this ticket you added only CMLCONNECT.
Are you sure you should be changing "mortgagestationconnect" ?
Is this an intended change?

Reply:
"mortgagestationconnect" css change is done as per Kimberly Giammarco comment for ticket MLC-3283.